### ISSUE TITLE
Submit 2.0 to PyPI

### ISSUE THREAD

**Issue by JohnDoee** [jaybaird/python-bloomfilter#9]
Can you submit 2.0 to PyPI ? seems like 1.1 is the latest: https://pypi.python.org/pypi/pybloom

**Comment by jaybaird** [comment]
I'll try to get to this this week. It was submitted to PyPI under an email I no longer have control of, so it may take me a while.

**Comment by bgilbert** [comment]
Any news on this?  It's sort of painful to have to use `dependency_links` in `setup.py` to declare a dependency on current pybloom, and pip 1.6 is apparently going to drop support for `dependency_links` altogether.

**Comment by Bachmann1234** [comment]
@jaybaird i'd be down for an update on this a well. I am interested in putting some work into this library if you are still able to maintain it.

**Comment by JohnDoee** [comment]
I think someone needs to take the baton, try and get in touch with @jaybaird and take over management of the project (PyPi stuff etc). It would be sad to see it die just because the current owner lost interest / time / whatever.

Judging by his twitter, he isn't internet-dead.

**Comment by Bachmann1234** [comment]
hmmm, ill reach out

**Comment by Bachmann1234** [comment]
@JohnDoee just sent @jaybaird an email offering to adopt this project.

**Comment by jaybaird** [comment]
Nope, not dead, just working at Apple :)

The PyPI thing is hard because I have no way of getting control of that package since it was submitted under my old work email. If you folks know of a way to get control of that again I'm all ears so I can submit.

**Comment by JohnDoee** [comment]
Good to see you alive and kicking :)

You can get your email changed by submitting a ticket here: http://sourceforge.net/p/pypi/support-requests/ at least, that's what other folks did.

**Comment by jaybaird** [comment]
Awesome, thanks mate. I'll get that in right now.

**Comment by bgilbert** [comment]
@jaybaird Any luck?  For some reason, I'm not seeing your ticket on the PyPI support board.

**Comment by jaybaird** [comment]
I had a busy weekend and had to get my SourceForge credentials back up to date as well. It’s on my to do list this week though.

Jay

**Comment by Bachmann1234** [comment]
@jaybaird any update? Anything we can do to help?

**Comment by hosford42** [comment]
@jaybaird Any word on this? I'm having the same issue... If there is no way to get access to the original account, could you republish under a new package name?

**Comment by jhgg** [comment]
+1

**Comment by mdshw5** [comment]
Yes, maybe this can be resubmitted under a new name? It's kind of silly that the PyPI version is so far behind the current master...

**Comment by JohnDoee** [comment]
Another issue is that it seems like jaybaird isn't very active here anymore.

My solution is that we should move it all to a fork and run the project there.

I elect myself to do all this, not because I'm the best choice but because someone has to put up their hand to get things moving.

Feel free to volunteer yourself to handle it or throw in some critique / comments. I'll give it a few weeks and just do it if nothing new appears.

**Comment by hosford42** [comment]
I considered doing that myself, but I didn't know if it would be polite. I am for it as well, if we don't get a response from him in that time frame. (It has been over a year since he was last seen here, after all.) If you need any assistance packaging, testing, or posting it, let me know.

**Comment by jhgg** [comment]
Honestly, if you are going to fork this, I'd recommend redoing the hash stuff too. Using cryptographically secure hashes in a bloomfilter is pointless & slows things down. Should be using murmur3 hash instead.

**Comment by jhgg** [comment]
I wrote https://github.com/jhgg/pybloof using the faster hashing algorithm. But I think it needs some review & API improvements.

**Comment by bgilbert** [comment]
[PyPI's draft transfer policy](https://docs.google.com/document/d/1elum7ENjQb0dLB4ATfYNtnXYVLUzsKacc0VWnHHJb2A/edit?usp=sharing) allows PyPI packages to be transferred by a PyPI admin with permission from the current owner.  If @jaybaird would be willing to give his permission, development could continue under the current PyPI package name (and presumably a different GitHub account).  If not, I think forking makes sense.

**Comment by RazerM** [comment]
I think I'll be submitting this to PyPI with a slightly different name, if only because I can't find any other implementations of a bloom filter that support Python 3 and/or Windows (which I plan on building wheels for using AppVeyor).

I'd also be happy to conribute the AppVeyor stuff if someone else would prefer to fork (@JohnDoee, @hosford42)

Edit: I looked at so many different bloom filters I forgot this one is pure Python, only need to upload...

**Comment by nirizr** [comment]
Looks like [this](https://github.com/joseph-fox/python-bloomfilter) fork is trying to reanimate this module

**Comment by joseph-fox** [comment]
@nirizr Yes, I have been trying my best to keep the forked repository active. 

It is available in [pypi](https://pypi.python.org/pypi/pybloom_live/).

**Comment by nirizr** [comment]
Yep. Just wanted to make it more visible :)
